src/front/ticketer/lib/backend_communication/logic/event/communication_event.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:convert';
+
+import 'package:dio/dio.dart';
+import 'package:ticketer/auth/auth.dart';
+import 'package:ticketer/auth/jwt_token.dart';
+import 'package:ticketer/backend_communication/logic/communication.dart';
+import 'package:ticketer/backend_communication/model/event.dart';
+import 'package:ticketer/backend_communication/model/response_codes.dart';
+import 'package:tuple/tuple.dart';
+
+class EventCommunication {
+  static const String _createEndPoint = "/event/create";
+  Future<Tuple2<Response, ResponseCode>> create(Event body) async =>
+      await BackendCommunication().postCallAuthorized(
+          _createEndPoint, Token(Auth().getCurrentAccount!.token),
```

Comment:
Add check if user is not logged in it will throw an error

Reply:
done